src/wallet/wallet.h

```diff
@@ -708,7 +708,7 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
     int64_t m_max_keypool_index = 0;
     std::map<CKeyID, int64_t> m_pool_key_to_index;
 
-    int64_t nTimeFirstKey = 0;
+    int64_t nTimeFirstKey GUARDED_BY(cs_wallet);
```

Comment:
Why remove the initialization?

Reply:
Ouch, that was not the intention. Thanks for noticing. Please re-review.